test/explorer/data_frame_test.exs

```diff
@@ -663,6 +663,16 @@ defmodule Explorer.DataFrameTest do
   end
 
   describe "mutate/2" do
+    test "with nil" do
+      df = DF.new(strs: ["a", "b", "c"], nums: [1, 2, 3])
+      assert DF.mutate(df, c: nil) |> inspect() == ~s(#Explorer.DataFrame<
+  Polars[3 x 3]
+  strs string ["a", "b", "c"]
+  nums s64 [1, 2, 3]
+  c null [nil, nil, nil]
+>)
```

Comment:
We should not rely on inspect for tests. We should use something like to_columns or assert on individual series. The tests below are a good example!

Reply:
what is preferred in `DF.dtypes(df1)` or `df1.dtypes`

Reply:
Any is fine. See the one we use the most in test and follow that template :)

Reply:
i have rewritten tests and pushed..